diff -up kscreenlocker-master/CMakeLists.txt.omv~ kscreenlocker-master/CMakeLists.txt
--- kscreenlocker-master/CMakeLists.txt.omv~	2023-05-20 22:51:36.597394919 +0200
+++ kscreenlocker-master/CMakeLists.txt	2023-05-20 22:51:42.136442727 +0200
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.16)
 project(kscreenlocker)
 set(CMAKE_C_STANDARD 99)
 set(PROJECT_VERSION "5.27.80")
-set(PROJECT_VERSION_MAJOR 5)
+set(PROJECT_VERSION_MAJOR 6)
 
 set(QT_MIN_VERSION "6.4.0")
 set(KF6_MIN_VERSION "5.240.0")
