diff -Naur kscreenlocker-5.5.3/greeter/fallbacktheme/LockScreen.qml kscreenlocker-5.5.3.tpg/greeter/fallbacktheme/LockScreen.qml
--- kscreenlocker-5.5.3/greeter/fallbacktheme/LockScreen.qml	2016-01-06 12:05:43.000000000 +0000
+++ kscreenlocker-5.5.3.tpg/greeter/fallbacktheme/LockScreen.qml	2016-01-22 23:31:09.329582822 +0000
@@ -46,7 +46,7 @@
     Image {
         id: background
         anchors.fill: parent
-        source: theme.wallpaperPathForSize(parent.width, parent.height)
+        source: ""/usr/share/mdk/backgrounds/OpenMandriva-splash.png""
         smooth: true
     }
 
