diff -up kscreenlocker-5.19.90/CMakeLists.txt.2~ kscreenlocker-5.19.90/CMakeLists.txt
--- kscreenlocker-5.19.90/CMakeLists.txt.2~	2020-09-17 17:52:07.026456284 +0200
+++ kscreenlocker-5.19.90/CMakeLists.txt	2020-09-17 18:00:36.765547823 +0200
@@ -59,6 +59,8 @@ find_package(KF5 ${KF5_MIN_VERSION} REQU
                     Solid
                     WindowSystem
                     XmlGui
+		    Plasma
+		    PlasmaQuick
                     )
 
 find_package(X11)
diff -up kscreenlocker-5.19.90/greeter/CMakeLists.txt.2~ kscreenlocker-5.19.90/greeter/CMakeLists.txt
--- kscreenlocker-5.19.90/greeter/CMakeLists.txt.2~	2020-09-17 13:04:47.000000000 +0200
+++ kscreenlocker-5.19.90/greeter/CMakeLists.txt	2020-09-17 17:52:07.026456284 +0200
@@ -42,6 +42,7 @@ target_link_libraries(kscreenlocker_gree
                         KF5::I18n
                         KF5::ConfigGui
                         KF5::Declarative
+                        KF5::PlasmaQuick
                         KF5::QuickAddons
                         KF5::WindowSystem
                         Qt5::Quick
