--- kscreenlocker-5.7.0/CMakeLists.txt.0001~	2016-07-04 19:21:25.888580235 +0200
+++ kscreenlocker-5.7.0/CMakeLists.txt	2016-07-04 19:22:41.705896207 +0200
@@ -45,6 +45,8 @@ find_package(KF5 ${KF5_MIN_VERSION} REQU
                     TextWidgets
                     WindowSystem
                     XmlGui
+		    Plasma
+		    PlasmaQuick
                     )
 
 find_package(X11)
--- kscreenlocker-5.7.0/greeter/CMakeLists.txt.0001~	2016-06-30 15:05:46.000000000 +0200
+++ kscreenlocker-5.7.0/greeter/CMakeLists.txt	2016-07-04 19:21:25.889580239 +0200
@@ -30,6 +30,7 @@ target_link_libraries(kscreenlocker_gree
                         KF5::I18n
                         KF5::ConfigGui
                         KF5::Declarative
+                        KF5::PlasmaQuick
                         KF5::QuickAddons
                         KF5::WindowSystem
                         Qt5::Quick
