diff -up kscreenlocker-master/greeter/fallbacktheme/LockScreen.qml.1~ kscreenlocker-master/greeter/fallbacktheme/LockScreen.qml
--- kscreenlocker-master/greeter/fallbacktheme/LockScreen.qml.1~	2023-08-02 12:40:16.740502962 +0200
+++ kscreenlocker-master/greeter/fallbacktheme/LockScreen.qml	2023-08-02 12:49:46.521459298 +0200
@@ -35,7 +35,7 @@ Item {
 
     Image {
         anchors.fill: parent
-        source: "file:" + PlasmaCore.Theme.wallpaperPathForSize(parent.width, parent.height)
+        source: "file:/usr/share/mdk/backgrounds/OpenMandriva-splash.png"
         smooth: true
     }
 
