diff -up kscreenlocker-6.2.0/autotests/pamtest.cpp.omv~ kscreenlocker-6.2.0/autotests/pamtest.cpp
--- kscreenlocker-6.2.0/autotests/pamtest.cpp.omv~	2024-10-12 04:57:57.972716319 +0200
+++ kscreenlocker-6.2.0/autotests/pamtest.cpp	2024-10-12 04:58:09.342832997 +0200
@@ -35,7 +35,7 @@ PamTest::PamTest()
 
 void PamTest::testLogin()
 {
-    PamAuthenticator auth("test_service", "test_user");
+    PamAuthenticator auth(QLatin1String("test_service"), QLatin1String("test_user"));
     QSignalSpy promptSpy(&auth, &PamAuthenticator::prompt);
     QSignalSpy promptForSecretSpy(&auth, &PamAuthenticator::promptForSecret);
     QSignalSpy succeededSpy(&auth, &PamAuthenticator::succeeded);
